### PR TITLE
Browser not supported page

### PR DESCRIPTION
**Pull request by elessar-ch**
This adds a browser not supported page and documentation of it.

### CODE REVIEW

**Comment by elessar-ch** [comment]
Have a look at the page here:
https://design.axa.com/feature-browser-not-supported/getting-started/browser-not-supported/

**Comment by elessar-ch** [comment]
Should we have the overlay on the toolkit as well?

File: docs/getting-started/browser-not-supported/index.hbs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    Meta Tags here.
+  </head>
+  <body>
+<!--[if lte IE 9]>
+{{#get snippets "/getting-started/browser-not-supported/snippets/full-page"}}{{{.}}}{{/get}}
+<![endif]-->
+    Your content for all your customers using modern browsers.
+  </body>
+</html>
+    {{/code-snippet}}
```
Comment (davidknezic) [inline docs/getting-started/browser-not-supported/index.hbs:84]: You don't have to break the indentation here

File: docs/getting-started/browser-not-supported/index.hbs [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: Browser not supported page
+description: asdf
```
Comment (davidknezic) [inline docs/getting-started/browser-not-supported/index.hbs]: How about?

> Notify users of old browsers to upgrade in order to enjoy your app.

File: docs/getting-started/browser-not-supported/index.hbs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    <p>
+      You can redirect to another URL that shows the user the
+      "Browser not supported" page using a meta tag.
+    </p>
+
+    {{#code-snippet "html"}}
+      <!DOCTYPE html>
+      <html>
+        <head>
+          <!--[if lte IE 9]><meta http-equiv="refresh" content="0; URL=/your/browser-not-supported-page.html"><![endif]-->
```
Comment (davidknezic) [inline docs/getting-started/browser-not-supported/index.hbs]: Detail, but by breaking this, you get shorter lines in the example:

``` html
<!--[if lte IE 9]>
  <meta http-equiv="refresh" content="0; URL=/your/browser-not-supported-page.html">
<![endif]-->
```

File: docs/getting-started/browser-not-supported/index.hbs [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+          Meta Tags here.
```
Comment (davidknezic) [inline docs/getting-started/browser-not-supported/index.hbs]: Just for copy-pasta's sake, I'd make this a comment. Also it gets highlighted in a more subtle way if it's a comment, showing users that it's only informational.

```
<!-- Meta Tags here. -->
```

File: scss/style/_browser-not-supported.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.browser-not-supported-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: $color-white;
+  overflow: scroll;
+  z-index: 999;
```
Comment (davidknezic) [inline scss/style/_browser-not-supported.scss]: Can you create a z-index variable for this one?

File: docs/getting-started/browser-not-supported/snippets/overlay.hbs [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+      <div class="header-main">
+        <div class="container">
+          <a href="#" class="header-brand">
+            <img class="header-brand-image" src="{{relative '/images/axa.svg'}}" />
+          </a>
+        </div>
+      </div>
+    </header>
+    <div class="container">
+      <div class="jumbotron">
```
Comment (davidknezic) [inline docs/getting-started/browser-not-supported/snippets/overlay.hbs]: Wondering if you could put the `container` into the `jumbotron`, so the blue background is full-width, like our other designs?

**Review by davidknezic — CHANGES_REQUESTED** [review]
Awesome! Very good explanation and great (IE-) intro 😄  Love that it can be used as a page and an overlay.

I've requested some minor changes before we can merge.

**Comment by davidknezic** [comment]
Just noticed this weird behavior...

img_2051

**Comment by elessar-ch** [comment]
Oki, fixed these issues @davidknezic.

**Review by davidknezic — APPROVED** [review]

**Pull request merged**